### ISSUE TITLE
Megahit for meta-transcriptome assembly

### ISSUE THREAD

**Issue by ARW-UBT** [voutcn/megahit#269]
Hello, 

I found a few contradicting posts in bioinformatics forums whether Megahit could be used or must not be used for metatranscriptome assemblies from microbiome meta-RNA-seq data. I have it a try but realized that the contigs, that several extrem long contigs were produced (e.g. over 100k bases). 

I think, there are no transcripts of this lenth in bacteria. So, what is you opinion on using Megahit for metatranscriptome analyses? Do you have special options that should be set in order to tell Megahit not to make assemblies as lon a possible?

**Comment by WJT0925** [comment]
I wonder if the eukaryotic metatranscriptome will not be affected by the overlapping peak, because the transcriptome lacks intron and intergenic region sequence, and the length of each gene normally assembled according to the overlapping peak is cut off, right?

**Comment by brauliovill** [comment]
> 

Did you ever figure out why your contigs were so long (>100k)? I'm getting the same issue

**Comment by ARW-UBT** [comment]
No, I did not figure out the reason for this, and I have not further inspected the content of the long assemblies. This would probably the next step to see if these long contigs are of 'biological' origin, or whether these are assembly artifacts. Insufficient removal of library adapters could cause such troubles...
